src/components/NewsletterForm/_newsletter-form.scss

```diff
@@ -43,6 +54,38 @@
   order: 4;
 }
 
+/* Errors */
+
+.newsletter-form__item-error {
+  background-color: #ffebeb;
```

Comment:
We should replace this with a WDS pallette colour

Reply:
Ticket added for this https://github.com/wellcometrust/corporate/issues/6018